turbo/jsonrpc/net_api.go

```diff
@@ -46,8 +46,17 @@ func NewNetAPIImpl(eth rpchelper.ApiBackend) *NetAPIImpl {
 }
 
 // Listening implements net_listening. Returns true if client is actively listening for network connections.
-// TODO: Remove hard coded value
-func (api *NetAPIImpl) Listening(_ context.Context) (bool, error) {
+func (api *NetAPIImpl) Listening(ctx context.Context) (bool, error) {
+	if api.ethBackend == nil {
+		// We're running in --datadir mode or otherwise cannot get the backend
```

Comment:
We have RemoteBackend in this case: which has GRPC methods

Reply:
Here: https://github.com/erigontech/erigon/blob/b2f772b4654e02037f8054ea2f9909586c18aa1a/cmd/rpcdaemon/cli/config.go#L355